lib/build.ts

```diff
@@ -119,14 +124,14 @@ export async function runBuildTask(
 	// api but it was broken until fixed in
 	// https://github.com/moby/moby/commit/7f334d3acfd7bfde900e16e393662587b9ff74a1
 	// which is why we check for 1.38 here
+	// The image must also have a v2 manifest so that docker won't try to default to x86.
 	const usePlatformOption: boolean =
 		!!task.dockerPlatform &&
```

Comment:
Is this calculation correct?  It seems to me that it is, but would like a verification.  
if updated docker AND there is a v2 manifest (which means docker knows the arch), then pass `--platform`.
Otherwise, if an old docker or no v2 manifest (way for docker to deterministically know the arch), then don't pass `--platform`

Reply:
Sounds right 👍